### DISCUSSION TITLE
Implementing Account Unlocker (capsolver, anticaptcha...)

### DISCUSSION THREAD

**Discussion by gniax** [mahrtayyab/tweety#178] [Ideas]
It would be interesting to add feature once the account is locked. You could use an existing solution like the one available at [https://github.com/ZakariaMQ/twitter-account-unlocker](https://github.com/ZakariaMQ/twitter-account-unlocker)

Alternatively, if there's an attribute within the Twitter API from Tweepy that allows us to act upon encountering the exception "To protect our users from spam...", that would be great. Or perhaps there's already a way to handle it that we could use. 

Any tips?

**Comment by mahrtayyab** [comment]
"To protect our users from spam..." usually raise `LoginDenied` Exception on Login.
I am already working on the solution. You are welcome to open PR if you can.

**Comment by mahrtayyab** [comment]
Added in latest update

**Reply by gniax** [reply]
awesome, thank you👍
